terraform/azure/unity_catalog.tfvars

```diff
@@ -1,13 +1,20 @@
 tenant_id = "9f37a392-f0ae-4280-9796-f1864a10effc"
 subscription_id = "3f2e4d32-8e8d-46d6-82bc-5bb8d962328b"
 
-reuse_rg = false
-rg_name = "unitycatalog"
-location = "West US"
+reuse_rg = true
```

Comment:
We could probably remove this file - or default it further

Reply:
I would leave this file as an example for inputs into the script